package.json

```diff
@@ -17946,7 +17946,7 @@
 		"update-emoji": "node ./scripts/generateEmojiShortcodeMap.mjs",
 		"update-licenses": "node ./scripts/generateLicenses.mjs",
 		"pretest": "pnpm run build:tests",
-		"vscode:prepublish": "pnpm run bundle"
+		"//vscode:prepublish": "pnpm run bundle"
```

Comment:
Same here

Reply:
yep, my bad. it's all because of pnpm issues on MacOS